### PR TITLE
Minor Autofill UI updates

### PR DESCRIPTION
**Pull request by amddg44**
Task/Issue URL: https://app.asana.com/0/0/1205435116192702/f, https://app.asana.com/0/0/1205495150811753/f, https://app.asana.com/0/0/1205495150811762/f, https://app.asana.com/0/0/1205533338187716/f
Tech Design URL:
CC:

**Description**:
Copy and layout updates resulting from "on by default" PFR

**Steps to test this PR**:

Note: Autofill is intentionally (and temporarily) enabled by default for the purposes of the Ship Review and I'll be reverting before merging, which will also fix this failing unit test `AppUserDefaultsTests.testDefaultAutofillStateIsFalse()`

1. In both empty state and when logins are present, confirm the label "Logins are stored securely on your device" is always visible as the section footer and screen looks similar to the figma design (note there was one [change](https://app.asana.com/0/0/1205541917560712/1205431733474528/f) requested from the design review -> the label is now indented) :
image
2. In the Autofill prompts, confirm the max width of the content is now restricted to 480px especially in iPhone landscape and on iPad as per figma:
image
3. In the Save Logins prompt, confirm the body copy is now `Logins are stored securely on your device in the Logins menu`
4. In the Password Generation prompt, confirm the title is now `Use a strong password from DuckDuckGo?`, the body is now `Passwords are stored securely on your device in the Logins menu.` and the CTA copy is `Use Strong Password`
5. Confirm the prompt to disable Autofill now triggers after 2 instead of 3 dismisses of the Save Login prompt (across 2 different sites)

**Copy Testing**:

* [ ] Use of correct apostrophes in new copy, ie `’` rather than `'`

**Orientation Testing**:

* [ ] Portrait
* [ ] Landscape

**Device Testing**:

* [ ] iPhone SE (1st Gen)
* [ ] iPhone 8
* [ ] iPhone X
* [ ] iPhone 14 Pro
* [ ] iPad

**OS Testing**:

* [ ] iOS 14
* [ ] iOS 15
* [ ] iOS 16

**Theme Testing**:

* [ ] Light theme
* [ ] Dark theme

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

File: DuckDuckGo.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1220,7 +1221,6 @@
 		317045BF2858C6B90016ED1F /* AutofillInterfaceEmailTruncatorTests.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = AutofillInterfaceEmailTruncatorTests.swift; sourceTree = "<group>"; };
 		31794BFF2821DFB600F18633 /* DuckUI */ = {isa = PBXFileReference; lastKnownFileType = wrapper; path = DuckUI; sourceTree = "<group>"; };
 		31951E8D2823003200CAF535 /* AutofillLoginDetailsHeaderView.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = AutofillLoginDetailsHeaderView.swift; sourceTree = "<group>"; };
-		31951E94282310FF00CAF535 /* WebsiteAccountExtension.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = WebsiteAccountExtension.swift; sourceTree = "<group>"; };
```
Comment (THISISDINOSAUR) [inline DuckDuckGo.xcodeproj/project.pbxproj:1234]: This file doesn't appear to be deleted, was this intentional?

File: DuckDuckGo/AppUserDefaults.swift [hunk 2]
```diff
@@ -192,7 +192,8 @@ public class AppUserDefaults: AppSettings {
             // In future, we'll use setAutofillCredentialsEnabledAutomaticallyIfNecessary() here to automatically turn on autofill for people
             // That haven't seen the save prompt before.
             // For now, whilst internal testing is still happening, it's still set to default to be enabled
-            return userDefaults?.object(forKey: Keys.autofillCredentialsEnabled) as? Bool ?? false
+            #warning("This is temporarily defaulted to true for ship review - undo before merging!")
+            return userDefaults?.object(forKey: Keys.autofillCredentialsEnabled) as? Bool ?? true
```
Comment (THISISDINOSAUR) [inline DuckDuckGo/AppUserDefaults.swift]: IF THIS COMMENT ISN'T MARKED AS OUTDATED, DO NOT MERGE

Don't mind me, just making sure 😂

File: DuckDuckGo/AutofillLoginSettingsListViewController.swift [hunk 3]
```diff
@@ -40,8 +40,9 @@ final class AutofillLoginSettingsListViewController: UIViewController {
     weak var delegate: AutofillLoginSettingsListViewControllerDelegate?
     weak var detailsViewController: AutofillLoginDetailsViewController?
     private let viewModel: AutofillLoginListViewModel
-    private let emptyView = AutofillItemsEmptyView()
+    private lazy var emptyView = AutofillItemsEmptyView()
```
Comment (THISISDINOSAUR) [inline DuckDuckGo/AutofillLoginSettingsListViewController.swift:43]: Is there a particular reason to change this one to lazy and not any of the others?

**Review by THISISDINOSAUR — COMMENTED** [review]
"In the Save Logins prompt, confirm the body copy is now Logins are stored securely on your device in the Logins menu"
Just to confirm this should only be for new users?

If so, behaviour wise LGTM, just have a couple of minor comments

File: DuckDuckGo.xcodeproj/project.pbxproj (hunk 1 above)
Comment (amddg44) [reply DuckDuckGo.xcodeproj/project.pbxproj:1234]: Ah sorry for confusion here - that file was moved to BSK as part of https://github.com/duckduckgo/iOS/pull/1911 but it wasn't removed from the .pbxproj so has been hanging around like a bad smell and I just couldn't take it anymore 😅   
image

File: DuckDuckGo/AutofillLoginSettingsListViewController.swift (hunk 3 above)
Comment (amddg44) [reply DuckDuckGo/AutofillLoginSettingsListViewController.swift:43]: The others are all immediately added as subviews as part of `viewDidLoad` -> `installSubviews` so didn't see any real benefit to making them lazy, while this view is now only added as a subview in one specific view state (`empty`)

**Comment by amddg44** [comment]
@THISISDINOSAUR I just realised I forget to answer this question 
> "In the Save Logins prompt, confirm the body copy is now Logins are stored securely on your device in the Logins menu"
Just to confirm this should only be for new users?

That should be for all users (we don't differentiate at all between new and existing users anymore in this prompt)

**Review by THISISDINOSAUR — APPROVED** [review]
I think I asked that because I thought I saw the wrong copy at one point, but I now can't replicate it, and think I might just have been looking at the screenshot above with the old copy.
So LGTM! (baring turning it to be off by default)

**Pull request merged**